prqlc/prql-compiler/src/sql/std.sql.prql

```diff
@@ -168,6 +201,12 @@ module mssql {
   @{binding_strength=11}
   let div_f = l r -> s"({l} * 1.0 / {r})"
 
+  # Mathematical functions
+  # https://learn.microsoft.com/en-us/sql/t-sql/functions/mathematical-functions-transact-sql
+  let ceil = column -> s"CEILING({column:0})"
+  let ln = column -> s"LOG({column:0})"
+  let pow = column exponent -> s"POWER({column:0}, {exponent:0})"
```

Comment:
So I guess we swap here... (sorry, should have spotted this before)

Then feel free to merge!

Reply:
Perfect I do that

Reply:
OK cool — so we should swap these in another PR, to `pow = exponent column`, so then `foo | pow 2` compiles to `POWER(foo, 2)`; is that consistent with what you were thinking?

Reply:
FK sorry I forgot the mssql overload ;( I'll fix that

Reply:
https://github.com/PRQL/prql/pull/3911